controller/src/main/java/org/jboss/as/controller/persistence/yaml/YamlConfigurationExtension.java

```diff
@@ -295,7 +300,13 @@ private void processAttribute(PathAddress address, ImmutableManagementResourceRe
                         OperationEntry operationEntry = rootRegistration.getOperationEntry(address, WRITE_ATTRIBUTE_OPERATION);
                         ModelNode op = createOperation(address, operationEntry);
                         op.get(NAME).set(attributeName);
-                        op.get(VALUE).set(processObjectAttribute((ObjectTypeAttributeDefinition) att, (Map<String, Object>) value));
+                        if (att instanceof MapAttributeDefinition) {
```

Comment:
I don't think this will handle ObjectMapAttributeDefinition correctly.

Reply:
@ehsavoie This review was also resolved without any information or code change. Could you add an explanation of the review and why there are no code changes involved?

Reply:
The change is in processMapAttribute